### PR TITLE
[hi] Localize continuous integration.md

### PR DESCRIPTION
**Pull request by Aayush987**
Hi,
This Pr is Hindi localization of content/hi/continuous_integration.md page.

ref : #270 
/lang hi

@anubha-v-ardhan @Garima-Negi and everyone plz review.

### CODE REVIEW

File: content/hi/continuous_integraion.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर एकीकरण
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर एकीकरण (Continuous_Integration) , जिसे  अक्सर CI कहा जाता है, कोड परिवर्तन को जल्दी से जल्दी एकीकृत करने का एक अभ्यास है। CI, [CD](/Continuous_Delivery/) पढ़ने के लिए आवश्यक है। ऐतिहासिक रूप से, CI की प्रक्रिया तब शरू होती है जब कोड परिवर्तन को सोर्स नियंत्रण सिस्टम (Git, Mercurial, या Subversal) में कमिट किया जाता है और वह परीक्षित उत्पाद, CD सिस्टम में जाने के लिए तैयार होता है।
```
Comment (sayantani11) [inline content/hi/continuous_integraion.md:9]: I feel like we shouldn't change the topic name to `निरंतर एकीकरण` as continuous Integration is a technical term

**Review by sayantani11 — APPROVED** [review]

File: content/hi/continuous_integraion.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/continuous_integraion.md:9]: Yeah, the `topic:`

Comment (Aayush987) [reply content/hi/continuous_integraion.md:9]: ok

Comment (Aayush987) [reply content/hi/continuous_integraion.md:9]: Done

File: content/hi/continuous_integraion.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर एकीकरण (Continuous_Integration)
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_integraion.md:2]:
```suggestion
title: निरंतर एकीकरण (Continuous Integration)

```

Comment (anubha-v-ardhan) [reply content/hi/continuous_integraion.md:2]: `_` seems unnecessary here

**Comment by anubha-v-ardhan** [comment]
Thank you @Aayush987 :)

**Comment by Aayush987** [comment]
> The file name should be `continuous_integration.md` and not `continuous_integraion.md`. I think we missed "t" in "integration" by mistake

OK, sorry my bad.

**Comment by anubha-v-ardhan** [comment]
Thank you @Aayush987 :)

**Pull request merged**